src/main/java/org/gridsuite/actions/server/dto/FilterMetaData.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.actions.server.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Schema(description = "Metadata on filters composing filter based contingency list")
+public class FilterMetaData {
+    @Schema(description = "filter uuid in filter data base")
+    private UUID id;
+
+    @Schema(description = "filter name")
+    private String name;
+
+    @Schema(description = "equipment type")
+    private String equipmentType;
```

Comment:
In my opinion, storing filter metadata on the actions server creates a risk: if a filter’s name or equipment type is updated, this information won’t stay synchronized.

These metadata should be enriched at service level when GET filter's info